dateparser/freshness_date_parser.py

```diff
@@ -11,7 +11,7 @@
 
 
 _UNITS = r'decade|year|month|week|day|hour|minute|second'
-PATTERN = re.compile(r'(\d+)\s*(%s)\b' % _UNITS, re.I | re.S | re.U)
+PATTERN = re.compile(r'(\d*\.?\d+)\s*(%s)\b' % _UNITS, re.I | re.S | re.U)
```

Comment:
We should probably support commas as well, for better multi-language support.

I think 1000+ values are unlikely in this scenario, so I’m personally OK with not taking into account the possibility of thousand separators messing things up.